src/Illuminate/Cache/Repository.php

```diff
@@ -35,7 +35,7 @@ class Repository implements CacheContract, ArrayAccess
     /**
      * The default number of minutes to store items.
      *
-     * @var int
+     * @var float
```

Comment:
I thought of float in docblocks as a general thing which means number, float, int, double, long double, long long int, short int. Guess I was wrong?